src/hb-ot-vorg-table.hh

```diff
@@ -84,9 +84,8 @@ struct VORG
     this->defaultVertOriginY = defaultVertOriginY;
     this->vertYOrigins.len = it.len ();
 
-    + it
-    | hb_apply ([c] (const VertOriginMetric& _) { c->copy (_); })
-    ;
+    for (const VertOriginMetric _ : it)
```

Comment:
Missing reference `&`.  Again, auto& maybe.  In all above ones as well.

Reply:
Or `const auto &` if that's a thing.

Reply:
Okay I see why you not referencing here.  It sucks that we cannot just use `const auto &` and let compiler handle.  Though, adding the const it might actually work.

Reply:
Compiler warnings related to it are brought https://github.com/harfbuzz/harfbuzz/pull/1985/commits/aa9960971a54bc1ae400d614bc7c63ec53cb8511 don't have the version with the flag to check it locally, it doesn't seem to.

Reply:
Happens with 'const auto &' also, https://circleci.com/gh/harfbuzz/harfbuzz/111243

Reply:
Okay.  But at least change most / all to `const auto` or `const auto &`?

Reply:
I can't get my mind around it, but what about `const auto &&`?

Reply:
Done cbc0646a6cfc0f6a04804d6888636f12130cd51b

Reply:
And reverted, clang didn't like that also :/